src/libs/actions/Report.js

```diff
@@ -181,76 +178,6 @@ function getSimplifiedIOUReport(reportData, chatReportID) {
     };
 }
 
-/**
- * Given IOU and chat report ID fetches most recent IOU data from DeprecatedAPI.
- *
- * @param {Number} iouReportID
- * @param {Number} chatReportID
- * @returns {Promise}
- */
-function fetchIOUReport(iouReportID, chatReportID) {
-    return DeprecatedAPI.Get({
-        returnValueList: 'reportStuff',
-        reportIDList: iouReportID,
-        shouldLoadOptionalKeys: true,
-        includePinnedReports: true,
-    }).then((response) => {
-        if (!response) {
-            return;
-        }
-        if (response.jsonCode !== 200) {
-            console.error(response.message);
-            return;
-        }
-        const iouReportData = response.reports[iouReportID];
-        if (!iouReportData) {
-            // IOU data for a report will be missing when the IOU report has already been paid.
-            // This is expected and we return early as no further processing can be done.
-            return;
-        }
-        return getSimplifiedIOUReport(iouReportData, chatReportID);
```

Comment:
This is still used in [syncChatAndIOUReports](https://github.com/Expensify/App/blob/dca62bd52329de8a26572eb60ca6fbb9ab793f63/src/libs/actions/Report.js#L942). We can remove it once `PayWithWallet` is refactored

Reply:
🎉